types/react-big-calendar/index.d.ts

```diff
@@ -73,7 +73,7 @@ export type DayLayoutAlgorithm = 'overlap' | 'no-overlap';
 export type NavigateAction = 'PREV' | 'NEXT' | 'TODAY' | 'DATE';
 export interface Event {
     allDay?: boolean | undefined;
-    title?: string | undefined;
+    title?: React.ReactNode;
```

Comment:
```suggestion
    title?: React.ReactNode | undefined;
```
just for sake of consistency, can we add `| undefined` as rest types have

Reply:
Since all of the items in Event have the question mark operator we could also potentially remove undefined from all of them.